### ISSUE TITLE
Benchmarks for individual Table operations reported on CI

### ISSUE THREAD

**Issue by radeusgd** [enso-org/enso#7306]
As suggested by @JaroslavTulach in #7270, we should be able to reliably track the performance of our Table operations before we start modifying them in hope of getting an improvement.

The goal of this ticket is to create benchmarks for basic Table operations and ensure that they are run daily on the CI alongside the other benchmarks and reported into the HTML reports generated by @Akirathan.

We need to do following:
- [ ] Adopt `Bench` _builder pattern_ as proposed by #7101 and rewrite all benchmarks to it
- [ ] Add a `Table_Benchmarks` project to the `test\` directory (@radeusgd)
	- [ ] Implement benchmarks for basic arithmetic operations (both on Long and Double storage, `+`, `-`, `* ` etc.)
	- [ ] Implement benchmarks for basic Text operations (`concat`, `length`, `starts_with`, `ends_with`, `contains`.
	- [ ] Ensure that all benchmarks run both for columns with no null values as well as 50% null.
	- [ ] Create a variant of arithmetic benchmarks when the column is backed by `MixedStorage` instead of `LongStorage` (can happen when we create a mixed column and then drop a few non-numeric values from it).
	- [ ] Implement benchmarks for 'functional' operations - `Column.zip`, `Column.map` using some Enso function (e.g. `Column.zip` with `[_, _]` or `_ * _` (like dot product), `Column.map (x-> "|" + x + "|")`).
	- This covers only part of the important capabilities. Other areas that should be benchmarked at some point is `Table.join`. This however should be done as a separate PR. 
	- There are some `aggregate` benchmarks already there. They are important as we are considering moving it to Enso (#5423, #6292) - before we do any of this we should likely revise these measurements. But this ticket in particular focuses mostly on the performance of basic column operations. What we can do is:
		- [ ] Ensure that the existing `aggregate` benchmarks are run alongside the other `Table` benchmarks and included in the summary.
- [ ] Ensure that `Table_Benchmarks` results are reported in our daily summary (@Akirathan)
	- [ ] Ensure that they are run on CI alongside other benchmarks.
	- [ ] Get their results saved in the same way as others and downloaded and summarized by the HTML reporting.

**Comment by jdunkerley** [comment]
Once we have the new benchmarking framework we should move the benchmarks in.
